registration-processor/pre-processor/registration-processor-cmd-validator-stage/src/main/java/io/mosip/registration/processor/stages/cmdvalidator/DeviceValidator.java

```diff
@@ -3,27 +3,43 @@
 import java.io.IOException;
```

Comment:
Since like the full class is changing, please create a unit test class and add all the unit test cases. Have atleast 2 test case for each type of validation +ve and -ve so in total atleast 6 test cases.

Reply:
I have added test cases in DeviceValidatorTests class.